src/MemberList.cpp

```diff
@@ -43,11 +43,12 @@ MemberListBackend::addUsers(const std::vector<RoomMember> &members)
     beginInsertRows(
       QModelIndex{}, m_memberList.count(), m_memberList.count() + (int)members.size() - 1);
 
-    for (const auto &member : members)
-        m_memberList.push_back(
-          {member,
-           ChatPage::instance()->timelineManager()->rooms()->currentRoom()->avatarUrl(
-             member.user_id)});
+    auto thisRoom = ChatPage::instance()->timelineManager()->rooms()->getRoomById(room_id_);
+    if (thisRoom.isNull())
```

Comment:
Well, now the insert is wrong!

Put the condition outside of the begin/endInsertRows and use a return :3

Reply:
Whoops, I missed a `!` 😬

Reply:
No, the bigger issue is that the count of your insert mismatches what you actually insert, which breaks Qt

Reply:
Well now. That might be a problem. I'll fix shortly.